.github/renovate.json

```diff
@@ -1,58 +1,28 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:best-practices",
```

Comment:
We don't need anything from it https://docs.renovatebot.com/presets-config/#configrecommended